### ISSUE TITLE
Missing `format` overrides

### ISSUE THREAD

**Issue by Geod24** [dlang-community/libdparse#135]
I have some generic code that generates visitors from all AST types, and as I was extending it I encountered this error:

```
./src/altered/visitor/ParseTree.d(122): Error: none of the overloads of 'format' are callable using argument types (const(AnonymousEnumDeclaration)), candidates are:
./submodules/libdparse/src/dparse/formatter.d(76):        dparse.formatter.Formatter!(Sink).Formatter.format(const(AddExpression) addExpression)
./submodules/libdparse/src/dparse/formatter.d(83):        dparse.formatter.Formatter!(Sink).Formatter.format(const(AliasDeclaration) aliasDeclaration, const(Attribute[]) attrs = null)
./submodules/libdparse/src/dparse/formatter.d(130):        dparse.formatter.Formatter!(Sink).Formatter.format(const(AliasInitializer) aliasInitializer)
./submodules/libdparse/src/dparse/formatter.d(147):        dparse.formatter.Formatter!(Sink).Formatter.format(const(AliasThisDeclaration) decl, const(Attribute[]) attrs = null)
./submodules/libdparse/src/dparse/formatter.d(161):        dparse.formatter.Formatter!(Sink).Formatter.format(const(AlignAttribute) alignAttribute)
./src/altered/visitor/ParseTree.d(122):        ... (189 more, -v to show) ...
```

**Comment by Geod24** [comment]
Actually, it's:
- AnonymousEnumDeclaration
- AnonymousEnumMember
- AutoDeclarationPart

**Comment by Basile-z** [comment]
Don't fix this one, rather make #346.
